pkg/client/cli/telepresence_test.go

```diff
@@ -1251,6 +1258,7 @@ func (hs *helmSuite) TestF_MultipleInstalls() {
 		hs.Contains(stdout, "with-probes: intercepted")
 	})
 	hs.Run("Uninstalls successfully", func() {
+		telepresence(hs.T(), "quit")
```

Comment:
this makes lots of sense, should we also add it to `TestZ_Uninstall`? It's not necessary right now, but if we ever add more tests between G and Z it could be

Reply:
Added.